src/test/java/hudson/plugins/locale/PluginImplTest.java

```diff
@@ -0,0 +1,66 @@
+package hudson.plugins.locale;
+
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.Before;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.LocalData;
+
+import java.util.Locale;
+
+import static org.junit.Assert.assertEquals;
+
+public class PluginImplTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    private PluginImpl plugin;
+
+    @Before
+    public void setUp(){
+        plugin = Jenkins.get().getExtensionList(PluginImpl.class).get(0);
+    }
+
+    @LocalData
```

Comment:
Why local data ? I don't see any xml file related to this test

Reply:
pardon me, I didn't use it at first, but the same error was coming up. But later while going through code and docs from the Jenkins test, I thought it might need some files to load. However, it did not resolve the issue.